build.gradle

```diff
@@ -1,3 +1,8 @@
+plugins {
+  id "org.owasp.dependencycheck" version "1.4.5"
+  id "com.github.johnrengelman.shadow" version "1.2.4"
```

Comment:
Then as per your findings, we cannot accept this change since our main source of repository is maven central not jcenter. :(